lib/server-builder.js

```diff
@@ -0,0 +1,154 @@
+import { SubProcess } from 'teen_process';
+import { fs, logger } from 'appium-support';
+import log from './logger';
+import B from 'bluebird';
+import _ from 'lodash';
+import path from 'path';
+import { EOL } from 'os';
+
+const IGNORED_ERRORS = [];
+const GRADLE_VERSION_KEY = 'gradle_version';
+const GRADLE_URL_PREFIX = 'distributionUrl=';
+const GRADLE_URL_TEMPLATE = 'https\\://services.gradle.org/distributions/gradle-VERSION-all.zip';
+const GRADLE_MAX_ERROR_LOG_LINES = 15;
+
+const VERSION_KEYS = [
+  GRADLE_VERSION_KEY,
+  'agp_version',
+  'compile_sdk_version',
+  'build_tools_version',
+  'min_sdk_version',
+  'target_sdk_version',
+  'kotlin_version'
+];
+
+const gradleLog = logger.getLogger('Gradle');
+
+class ServerBuilder {
+  server_versions
```

Comment:
what is the purpose of this variable?

Reply:
this is just explicit property declaration for syntax highlighting purposes. I suppose Appium style is to not declare them, so I'll remove it and review my code for similar small issues.

Reply:
we simply try to avoid static variables. Moving it to module level could be a good idea. We could also use LRU cache to make sure the value is never going to be too large